### PR TITLE
Feature/remove select one #169929680

### PR DESCRIPTION
**Pull request by LoganArnett**
This is for ticket [#169929680](https://www.pivotaltracker.com/story/show/169929680)

### CODE REVIEW

File: app/javascript/utils/formUtils.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ export const maxLengthMap = {
   income: 16
 }
 
+export const reorderEmptyOptions = (options) => {
+  options.some((option, i) => isEmpty(option.value) && options.unshift(option) && options.splice(i + 1, 1))
```
Comment (LoganArnett) [inline app/javascript/utils/formUtils.js]: Loop through and find an empty value, unshift it to the front, and splice the old index out

**Review by akegan — COMMENTED** [review]
QA'd and looked at code and looks good. Only concern is that it looks like snapshot changes from the react-final-form update bled into this PR.

Semaphore is also failing because of the update to a higher version of ruby in the other PR. We can get that one in first.

**Review by akegan — APPROVED** [review]
approving because I think this'll be good to go once semaphore passes, please just keep an eye on the snapshots and make sure the changes make sense

**Comment by akegan** [comment]
@LoganArnett this is good to merge!

**Pull request merged**